lib/srv/db/access_test.go

```diff
@@ -2011,7 +2011,9 @@ func withRDSPostgres(name, authToken string) withDatabaseOption {
 				Region: testAWSRegion,
 			},
 			// Set CA cert, otherwise we will attempt to download RDS roots.
-			CACert: string(testCtx.databaseCA.GetActiveKeys().TLS[0].Cert),
+			TLS: types.DatabaseTLS{
+				CACert: string(testCtx.databaseCA.GetActiveKeys().TLS[0].Cert),
```

Comment:
Huh, strange that we have CACert in both DatabaseSpecV3 and in DatabaseSpecV3.TLS - why did it need to be changed to use the latter?

Reply:
`CACert` should deprecated after introducing the `TLS` block. Current logic has some backward compatibility but I guess we will remove `CACert` at some point. 
https://github.com/gravitational/teleport/blob/69197efb1d92c135f02b0315df8ff70edb44618f/api/proto/teleport/legacy/types/types.proto#L292-L298

Reply:
Ah, thanks for the explanation I was just curious so I'll mark this resolved